pallets/registrar/src/benchmarks.rs

```diff
@@ -106,14 +147,15 @@ mod benchmarks {
     }
 
     #[benchmark]
-    fn register_with_relay_proof(x: Linear<5, 3_000_000>, y: Linear<1, 50>, z: Linear<1, 10>) {
-        let mut data = vec![];
-        // Number of keys
-        for _i in 1..z {
-            data.push((b"code".to_vec(), vec![1; (x / z) as usize]).into())
-        }
-
-        let storage = new_genesis_data(data);
+    fn register_with_relay_proof(
+        x: Linear<100, 3_000_000>,
+        z: Linear<1, 10>,
+    ) -> Result<(), BenchmarkError> {
+        // This extrinsic is disabled in flashbox runtime, return 0 weight there
+        let _origin = T::RegisterWithRelayProofOrigin::try_successful_origin()
+            .map_err(|_| BenchmarkError::Weightless)?;
```

Comment:
I can't use this origin below because the signature is only valid for the account created by `create_funded_user` below, but if this returns an error it means that the extrinsic cannot be called, so we return a weight of zero.

Reply:
nice! then the benchmarks dont fail right?

Reply:
Correct, I have run them and there is a weight of 0 in flashbox, and non-zero in dancebox